mlir/lib/Dialect/QCO/Transforms/Decomposition/UnitaryMatrices.cpp

```
@@ -0,0 +1,229 @@
+    const double lambda = gate.parameter[1];
+    return u2Matrix(phi, lambda);
+  }
+  if (gate.type == GateKind::H) {
+    return H_GATE;
+  }
+  llvm::reportFatalInternalError(
+      "unsupported gate type for single qubit matrix");
+}
+
+// TODO: remove? only used for verification of circuit and in unittests
+Eigen::Matrix4cd getTwoQubitMatrix(const Gate& gate) {
```

Cppcheck (2.20.0)

[style] The function 'getTwoQubitMatrix' is never used.

(unusedFunction)